model/event/event_registry.go

```diff
@@ -87,38 +77,36 @@ func (r *eventRegistry) IsSubscribable(resourceType, eventType string) bool {
 	return r.isSubscribable[e]
 }
 
-func NewEventFromType(resourceType string) interface{} {
-	registry.lock.RLock()
-	defer registry.lock.RUnlock()
+func (r *eventRegistry) setNeverExpire(resourceType, eventType string) {
```

Comment:
nit: should be neverExpires to match variable name, although I'm wondering if it would be reasonable to switch to `unexpirable` to match the language we use for hosts elsewhere in the codebase.

Reply:
I changed it to unexpirable